src/Word.java

```diff
@@ -1,78 +1,63 @@
 import java.util.HashSet;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 public class Word {
-    private Set<Character> guessedChars;
-    private String originalWord;
 
-    public Word(String originalWord) {
-        this.originalWord = originalWord;
-        guessedChars = new HashSet<>();
-    }
-
-    private static Set<Character> unguessableChars = new HashSet<>();
-    {
-        unguessableChars.add(' ');
-        unguessableChars.add('"');
-        unguessableChars.add('.');
-        unguessableChars.add(',');
-        unguessableChars.add('@');
-    }
+  private Set<Character> guessedChars;
+  private String originalWord;
 
-    public GuessResult guess(String guess) {
-        if (guess.length() == 1) {
-            char c = guess.charAt(0);
+  Word(String originalWord) {
+    this.originalWord = originalWord;
+    guessedChars = new HashSet<>();
+  }
 
-            if (unguessableChars.contains(c)) {
-                return GuessResult.UNGUESSABLE_CHAR;
-            }
+  public int wordLength(){
+    return (int) originalWord.chars().mapToObj(i -> (char) i).filter(Character::isLetter).count();
```

Comment:
Why do you need this? Also, it's not clear from the function name that you're filtering out all non-letters.